server/gitlab/api.go

```diff
@@ -412,3 +412,35 @@ func (g *gitlab) ResolveNamespaceAndProject(
 	}
 	return "", "", ErrNotFound
 }
+
+func (g *gitlab) TriggerNewBuildPipeline(user *UserInfo, repo interface{}, refToCommit *string) (*internGitlab.Pipeline, error) {
```

Comment:
I know that `client.Pipelines.CreatePipeline` accepts `repo` as `string` and `int`. Maybe we just stick with `string` for simplicity?

Reply:
Current code looks fine: https://github.com/lufty-demansol/mattermost-plugin-gitlab/blob/master/server/gitlab/api.go#L416